### ISSUE TITLE
Detect vulnerable core modules too?

### ISSUE THREAD

**Issue by JRaspass** [vti/cpan-audit#8]
Suppose a cpanfile had this https://metacpan.org/pod/Carton#PERL-VERSIONS
```
requires 'perl', '5.16.3';
```

Using https://metacpan.org/release/Module-CoreList we should be able to detect that Compress::Raw::Zlib is vulnerable even though it was never explicitly installed, maybe some kind of maybe vulnerable if you actually use it category.

**Comment by vti** [comment]
Yeah, and by default we can use the current perl's version. I would appreciate a pull request 😊

**Comment by JRaspass** [comment]
SGTM I'l add it to my never ending todo list ;-)

I'll see what I can come up with this week, thanks for the quick reply :-)
